spec/controllers/registrations_controller_spec.rb

```diff
@@ -48,6 +48,10 @@
           expect(subject).to receive(:clean_up_passwords)
           post :create, user: user_attributes
         end
+
+        it "should create a new ZooniverseUser" do
+          expect(ZooniverseUser.where(login: login).first).to_not be_nil
```

Comment:
Database cleaner isn't cleaning the sqllite db and some records are being left around, hence this passes without a post request to create a resource.